src/pas/plugins/authomatic/patches/authomatic.py

```diff
@@ -0,0 +1,147 @@
+from authomatic.exceptions import FetchError
+from authomatic.providers import BaseProvider
+from http import client as http_client
+from pas.plugins.authomatic.logging import logger
+from urllib import parse
+
+import authomatic.core
+import ssl
+
+
+def patch_base_provider_fetch():
```

Comment:
@ericof Could you please add a comment about the purpose of the patch / how we would know if it can be removed in the future?